lib/JITIDL/JITTypes.h

```diff
@@ -10,9 +10,19 @@ import "wtypes.idl";
 #endif
 
 #if defined(_M_IX86) || defined(_M_ARM)
+#ifdef __midl
+#define CHAKRA_WB_PTR int
+#else
+#define CHAKRA_WB_PTR void*
+#endif
 #define CHAKRA_PTR int
 #define BV_SHIFT 5
 #elif defined(_M_X64) || defined(_M_ARM64)
+#ifdef __midl
+#define CHAKRA_WB_PTR __int64
+#else
+#define CHAKRA_WB_PTR void*
+#endif
```

Comment:
What's the reason for this change... Some fields need WB in the IDL headers structures? Because those structures were WB recycler allocated?